docs/farming-&-staking/farming/additional-guides/gpu-plotter.md

```diff
@@ -8,79 +8,94 @@ keywords:
     - GPU
 ---
 
-Official builds of the **GPU Plotter** are now available!
-
 ## Overview
 
-Autonomys requires a portion of your drive(s) to be reserved for plots (SSD or NVMe). Once plotted, these plots are farmed using your CPU. This farming process is not CPU intensive, and most modern consumer CPUs can farm a very large amount of drive space. The initial creation of these plots (and replotting) is much more CPU intensive, generally making the CPU the largest bottleneck in creating these plots.
+Autonomys utilizes your drive storage, specifically SSD or NVMe drives, to store plots. After the plotting process is finished, these plots are then farmed using your CPU. Farming is not particularly demanding on the CPU, enabling most modern processors to manage a substantial farm size. However, the plot creation process is highly resource intensive, which makes CPU plotting the main bottleneck.
 
-With the release of GPU plotting, you can now use your supported GPU(s) to create these plots, either in addition to or instead of using the CPU. While many newer CPUs can plot a sector in less than 2 minutes, a high end GPU can do it in less than 6 seconds.
+Utilizing GPU plotting allows you to harness the power of compatible GPUs for plot generation and replotting, either in conjunction with or as a substitute for CPU processing. While many modern CPUs can complete the plotting of a sector in less than two minutes, a single high performance GPU can accomplish the same task in under five seconds, greatly improving efficiency and speed.
 
-GPU plotting is not required, but it is much more energy efficient and significantly faster than using a CPU alone.
+Although GPU plotting is not mandatory, it provides enhanced energy efficiency and speed compared to relying solely on a CPU.
 
 
 :::note Plot Format Compatibility
 GPU plotting employs the new **v1 plot format**, which is applicable to any plots created with versions released on or after **July 5th**. In contrast, older software versions generated plots in the **v0 format**, which is only compatible with CPU plotting.
 :::
 
-### Supported Platforms
-| Platform          |    Linux    |   Windows   |
-|-------------------|:-----------:|:-----------:|
-| Subspace          |     ✅      |     ✅      |
-| Space Acres       |     ✅      |     ✅      |
+
+### Supported Platforms and Graphics Card Compatibility
+| Platform          | 🐧 Linux | 🪟 Windows | [Nvidia](#nvidia) | [AMD](#amd) | [Intel](#intel) |
+|-------------------|:-------:|:---------:|:---------:|:-----------:|:------------:|
+| [Subspace](https://github.com/autonomys/subspace/releases)          |   ✅    |    ✅     |    ✅     |     🔜      |      🔜      |
```

Comment:
```suggestion
| [Subspace CLI](https://github.com/autonomys/subspace/releases)          |   ✅    |    ✅     |    ✅     |     🔜      |      🔜      |
```

Reply:
Good catch! I’ve updated it to 'Advanced CLI' to align with the naming convention used throughout the documentation.